skimage/metrics/set_metrics.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+from ._set_metrics import hausdorff_distance_onesided
+
+
+def hausdorff_distance(a, b):
+    """
+    Calculate the Hausdorff distance [1]_ between two sets of points.
+
+    The Hausdorff distance is the maximum distance between any point on
+    ``a`` and its nearest point on ``b``, and vice-versa.
+
+    Parameters
+    ----------
+    a, b : ndarray, dtype=bool
```

Comment:
I'd remove the dtype specification, since this works totally fine for integer arrays. It could be used to compare two segmentations, for example.

Reply:
Done